AirCasting/Utils/SystemSettings.swift

```diff
@@ -0,0 +1,32 @@
+// Created by Lunar on 15/07/2021.
+//
+
+import Foundation
+import SwiftUI
+
+protocol SettingsRedirectionProtocol {
+    func goToLocationAuthSettings()
+    func goToBluetoothAuthSettings()
+}
+
+final class SettingsRedirection: SettingsRedirectionProtocol, ObservableObject {
```

Comment:
I think `ObservableObject` is not needed here since there is no state whatsoever

Reply:
![Screenshot 2021-07-22 at 10 36 30](https://user-images.githubusercontent.com/44276070/126611367-5ac16539-8652-46e2-a444-cc8b5f448f4f.png)
I think it is needed here to use EnviromentObject but maybe I am wrong 😳

Reply:
Ok, next comments changes a bit